lib/ansible/executor/module_common.py

```diff
@@ -786,20 +786,25 @@ def _find_module_utils(module_name, b_module_data, module_path, module_args, tas
         lines = b_module_data.split(b'\n')
         module_names = set()
 
-        requires_module_list = re.compile(r'(?i)^#requires \-module(?:s?) (.+)')
+        requires_module_list = re.compile(r'(?i)^#(?:s*)requires \-module(?:s*) (Ansible\.ModuleUtils\..+)')
```

Comment:
Why do we need the `(?:s*)` part in this regex in the beginning? Do we want to support -module and -modules?